src/grocker/__main__.py

```diff
@@ -74,6 +74,10 @@ def file_path_or_none_type(x):
         '--docker-image-prefix', metavar='<url>',
         help='docker registry or account on Docker official registry to use'
     )
+    parser.add_argument(
+        '--image-base-name', metavar='<name>',
+        help='base name for the image (default is based on pip metadata)'
```

Comment:
You should explain which part of the image name you replace (e.g ``<docker-image-prefix>/<base-image-name>:<image-version>``)

nit: missing comma